Sources/Basics/Observability.swift

```diff
@@ -134,6 +143,16 @@ public final class ObservabilityScope: DiagnosticsEmitterProtocol {
         self.diagnosticsHandler.handleDiagnostic(scope: self, diagnostic: diagnostic)
     }
 
+    // metrics
+
+    public func makeTimer(label: String) -> Timer {
```

Comment:
I wonder if this would be a time to pass in a start time.  It's not always `Date.now` since the metrics might come from a subsystem, but that seems like the default.  And then when "ending" the recording (the way I'm thinking about this) it would also allow a end time (or duration) but default it to `Date.now`.